app/views/unattended/provisioning_templates/snippet/puppetlabs_repo.erb

```diff
@@ -41,18 +41,15 @@ if host_param_true?('enable-puppetlabs-repo')
   repo_subdir = ''
```

Comment:
remove this repo_subdir, too?

Reply:
done - I was going to leave that for when I pulled the whole puppet5 release from the templates as it looked like the generic puppetlabs-release may have had some depends on a a subdir, but as puppet5 is EOL I guess it doesn't matter, and it looks like that may have been incorrect when I first looked it up due to symlinks